tools/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -340,7 +348,7 @@ def update_source_file(file_id):
 
 
 def plist_to_html(file_path, output_path, html_builder,
-                  skip_report_handler=None):
+                  skip_report_handler=None, trim_path_prefixes=None):
```

Comment:
You do not need to set `trim_path_prefixes` to `None` here. It is already set as a default value in `get_report_data_from_plist`.

Reply:
I added the new argument `trim_path_prefixes` after `skip_report_handler` to minimize the impact of such signature change. To define `trim_path_prefixes` as non-optional, I have to move it before `skip_report_handler` (which was already optional). Do you prefer that ?

Reply:
I think we can keep it as it is in your last commit.